docs/apache-airflow/core-concepts/executor/index.rst

```diff
@@ -77,4 +77,4 @@ There are two types of executor - those that run tasks *locally* (inside the ``s
 
 .. note::
 
-    Something that often confuses new users of Airflow is that they don't need to run a separate ``executor`` process. This is because the executor's logic runs *inside* the ``scheduler`` process - if you're running a scheduler, you're running the executor.
+    Something that often confuses new users of Airflow is that they don't need to run a separate ``executor`` process using one of the Local Executors or Remote Executors (because executor's logic runs *inside* the ``scheduler`` process and this logic will run the tasks locally or not depending the executor selected).
```

Comment:
```suggestion
    New Airflow users may assume they need to run a separate executor process using one of the Local or Remote Executors. This is not correct since the executor logic runs *inside* the scheduler process, and will run the tasks locally or not depending the executor selected.
```

I needed to read the first part of the sentence several times to understand. Flipping the conditional may be useful…

Reply:
thanks for the review